pkg/scheduler/apis/config/types.go

```diff
@@ -96,6 +96,11 @@ type KubeSchedulerConfiguration struct {
 	// Extenders are the list of scheduler extenders, each holding the values of how to communicate
 	// with the extender. These extenders are shared by all scheduler profiles.
 	Extenders []Extender
+
+	// MaintainCacheWhenNonLeader enables caching for non-leader candidates. This makes non-leader candidates
```

Comment:
s/candidates/replicas

Reply:
I am okay with the suggestion. Just I noticed https://github.com/kubernetes/kubernetes/blob/bceff79876ebd33cdc108300ba715b12bf12a0e4/staging/src/k8s.io/component-base/config/types.go#L46 use `candidates`, should we use `candidates` here consistently?

Reply:
ah ok, I didn't know that was the word already used. Let's keep the same.